src/field/index.js

```diff
@@ -4,15 +4,31 @@ import trim from 'trim';
 import OptionSelectionPane from './option_selection_pane';
 import * as l from '../core/index';
 
-export function setField(m, field, value, validator = str => trim(str).length > 0, ...args) {
+const minMax = (value, min, max) => value.length >= min && value.length <= max;
+
+const getDefaultValidator = field => {
+  switch (field) {
+    case 'family_name':
+      return str => minMax(trim(str), 1, 150);
+    case 'given_name':
+      return str => minMax(trim(str), 1, 150);
+    case 'name':
```

Comment:
Nitpick: You could write this as:

```javascript
switch (field) {
  case 'family_name':
  case 'given_name':
    return str => minMax(trim(str, 1, 150);
```

etc.

Reply:
fixed